src/Cdn/Cdn/Helpers/ModelExtensions.cs

```diff
@@ -589,7 +675,7 @@ public static DeliveryRule ToDeliveryRule(this PSDeliveryRule psDeliveryRule)
                 Name = psDeliveryRule.Name,
                 Order = psDeliveryRule.Order,
                 Actions = psDeliveryRule.Actions.Select(action => action.ToDeliveryRuleAction()).ToList(),
-                Conditions = psDeliveryRule.Conditions.Select(condition => condition.ToDeliveryRuleCondition()).ToList()
+                Conditions = psDeliveryRule.Conditions?.Select(condition => condition.ToDeliveryRuleCondition()).ToList()
```

Comment:
Is it a bug fix or previously Conditions cannot be null?

Reply:
This is a bug fix. Conditions should be nullable.